extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -8,15 +8,18 @@ define('ext.wikia.adEngine.provider.gpt.googleTag', [
 	'use strict';
 
 	var logGroup = 'ext.wikia.adEngine.provider.gpt.googleTag',
+		cmd = [],
 		registeredCallbacks = {},
 		slots = {},
 		slotQueue = [],
 		pageLevelParams,
-		googleTag,
 		pubAds;
 
-	function GoogleTag() {
+	function GoogleTag(storeCommands) {
 		this.initialized = false;
+		this.storeCommands = storeCommands;
+		slots = {};
+		slotQueue = [];
```

Comment:
Do we have to reset `slots` and `slotQueue` in the constructor? They aren't "properties" of an instance and we define them with assignment above. I wonder if we can get rid of one or another assignment :japanese_ogre: